src/resolve/resolved-dns-rr.c

```diff
@@ -826,6 +827,105 @@ static char *format_txt(DnsTxtItem *first) {
         return s;
 }
 
+static char *format_svc_param_value(DnsSvcParam *i) {
+        _cleanup_free_ char *value = NULL;
+
+        switch (i->key) {
+        case DNS_SVC_PARAM_KEY_ALPN: {
+                size_t sz = 0, offset = 0;
```

Comment:
move `size_t` declaratio inside the loop, i.e. we usually want that variables are defined at innermost scope (except for `int r`, that always gets function scope)

Reply:
Applied in #30661.